addons/mail/tests/test_res_partner.py

```diff
@@ -185,3 +185,99 @@ def test_res_partner_merge_wizards(self):
         all_msg = p2_msg_ids_init + p1_msg_ids_init + p1_msg1
         self.assertEqual(len(p2.message_ids), len(all_msg) + 1, 'Should have original messages + a log')
         self.assertTrue(all(msg in p2.message_ids for msg in all_msg))
+
+    def test_res_partner_get_starred_count_with_empty_body(self):
```

Comment:
Would maybe be better to have a single test that updates the message, to simplify a bit them and have a bit less code. You have a lot of duplicated boilerplate :) .